tests/dist/python/run_dist_objects.py

```diff
@@ -88,18 +145,21 @@ def test_dist_tensor(g, rank):
 elif mode == "client":
     os.environ['DGL_NUM_SERVER'] = str(num_servers_per_machine)
     dgl.distributed.initialize(ip_config, net_type=net_type)
-    global_rank = dgl.distributed.get_rank()
 
     gpb, graph_name, _, _ = load_partition_book(graph_path + '/{}.json'.format(graph_name), part_id, None)
     g = dgl.distributed.DistGraph(graph_name, gpb=gpb)
-    target = os.environ.get('DIST_DGL_TEST_OBJECT_TYPE', 'DistTensor') 
-    if target == "DistTensor":
-        test_dist_tensor(g, global_rank)
-    elif target == "DistEmbedding":
-        # TODO: implement DistEmbedding
-        pass
+
+    target_func_map = {"DistTensor": test_dist_tensor,
+                       "DistEmbedding": test_dist_embedding,
+                       }
+
+    target = os.environ.get("DIST_DGL_TEST_OBJECT_TYPE", "")
+    if target not in target_func_map:
```

Comment:
why do we need this `if-else`? optionally run some tests during debug? I think it's better to define in `@pytest.mark.parametrize()` or just remove such arg.

Reply:
This file is not run by `pytest`, so we need to do that